### ISSUE TITLE
sandbox on github but not svn

### ISSUE THREAD

**Issue by lgatto** [ComputationalProteomicsUnit/Pbase#11]
Currently, we have a `sandbox` directory on github, which is useful to share in progress scripts. There is more in my local dir, but that's fine, as there is not need to clutter too much. `sandbox` also lives partially on the Bioconductor [svn server](https://hedgehog.fhcrc.org/bioconductor/trunk/madman/Rpacks/Pbase/), which is not useful at all. Fortunately, we have `sandbox` (an more) in `.Rbuildignore`, but I would like to get completely rid of it on the svn server. 

Is that fine for you, @sgibb? Do you know what the best way would be to set that in place?

**Comment by sgibb** [comment]
I agree, we don't need the sandbox directory on the bioc svn server. Is `svn propedit svn:ignore .` useful? (I have no idea whether original svn commands work in a git-svn setup.)

**Comment by lgatto** [comment]
That does not seem to word. Even when ignoring works using `svn` (in a `svn checkout` dir), the directory gets added back by `git-svn`. 

From the [git-svn manual](https://www.kernel.org/pub/software/scm/git/docs/git-svn.html)
>  We ignore all SVN properties except svn:executable.

**Comment by lgatto** [comment]
Absolutely. I had forgotten about this issue.
